src/main.cpp

```diff
@@ -52,9 +52,18 @@ extern void qt_set_sequence_auto_mnemonic ( bool bEnable );
 #ifndef SERVER_ONLY
 #    include "clientrpc.h"
 #endif
+#include "cmdline.h"
 
 // Implementation **************************************************************
 
+QString UsageArguments ( char* argv );
+
+void OnFatalError ( QString errMsg )
+{
+    qCritical() << qUtf8Printable ( errMsg );
+    exit ( 1 );
+}
+
```

Comment:
Note that OnFatalError could be used on many places in main.cpp.

```
    qCritical() << qUtf8Printable ( ... );
    exit ( 1 );
```

Is used in many places...